### PR TITLE
Update profile page for MCR

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
Makes the profile page for MCR quite snazzy.

Preview:
Screen Shot 2022-07-18 at 2 31 11 PM

### How to test
./dev local
Click on My Account -> Manage Account!

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by karla-vm — APPROVED** [review]
re: "I don't necessarily like this. Any thoughts on how to improve an inline link with the format we've got going for these pages?"

I don't have any ideas on this, but otherwise it's looking good!

File: services/ui-src/src/routes/Profile/Profile.tsx [hunk 1]
```diff
@@ -19,17 +21,29 @@ export const Profile = () => {
...
+        <span>
+          <Link
+            sx={sx.emailText}
+            href={createEmailLink(intro.email)}
+            target="_blank"
+          >
+            {intro.email.address}
+          </Link>
+        </span>
+        {"."}
```
[the remark this answers is not in the document]
Comment (braxex) [reply services/ui-src/src/routes/Profile/Profile.tsx:44]: Nothing wrong with the way you've done it. I'm guessing you followed the pre-existing stuff in `EmailCard`. 👍 

I've made a couple suggestions:
- Can remove the unneeded `<span>` elements
- Can remove brackets around period

Comment (braxex) [inline services/ui-src/src/routes/Profile/Profile.tsx:44]:
```suggestion
          <Link
            sx={sx.emailText}
            href={createEmailLink(intro.email)}
            target="_blank"
          >
            {intro.email.address}
          </Link>
        .
```

File: services/ui-src/src/routes/Profile/Profile.tsx [hunk 2]
```diff
@@ -51,11 +65,17 @@ const sx = {
       marginTop: "3.5rem",
     },
   },
-  heading: {
-    marginBottom: "2rem",
+  headerText: {
+    marginBottom: "rem",
```
Comment (braxex) [inline services/ui-src/src/routes/Profile/Profile.tsx:69]: Still need the bottom rem on this page to push the footer down.

```suggestion
    marginBottom: "2rem",
```

**Review by braxex — CHANGES_REQUESTED** [review]

**Review by braxex — APPROVED** [review]

**Pull request merged**